app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_split = command.split()
+    file_in_name = file_split[1]
+    file_out_name = file_split[2]
```

Comment:
Save split result to a separate variable

Validate that the command length is equal exactly to 3 arguments

Validate that command name is exactly "cp"

Use unpacking to access command parts